apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/util/DataSampler.java

```diff
@@ -19,11 +19,20 @@
  * @param <T> The type of data being stored.
  */
 public class DataSampler<T> implements Iterable<T> {
+  /** Private field to set hashmap that will be used for the dataSet. */
```

Comment:
Don't need to mention its private. Assume all the information about access modifiers, type, and defaults are assumed. We ideally want the documentation to tell us what we don't see below, (such as "what is this for?" if its not obvious)

Reply:
done